openpdf/src/main/java/com/lowagie/text/pdf/PdfContentParser.java

```diff
@@ -84,16 +86,18 @@ public PdfContentParser(PRTokeniser tokeniser) {
      * @return the same <CODE>ArrayList</CODE> given as argument or a new one
      * @throws IOException on error
      */    
-    public ArrayList parse(ArrayList ls) throws IOException {
-        if (ls == null)
-            ls = new ArrayList();
-        else
+    public List<PdfObject> parse(List<PdfObject> ls) throws IOException {
```

Comment:
This looks to be a breaking change. I would recommend adding this as a new method signature that users can opt in to. Perhaps in the future we can release a version that includes a bunch of breaking changes, but we've strived to avoid that (for now).

Reply:
Yeah. If required, I'll look through all my changes from this PR and return old methods back deprecating them.

Reply:
@kulatamicuda @rtfarte Have all this been fixed now?

Reply:
@andreasrosdal I still have not pushed this one. Done something like 80% and will finish in day or two.

Reply:
Addressed in https://github.com/LibrePDF/OpenPDF/pull/177